frontend/src/component/admin/network/NetworkTrafficUsage/NetworkTrafficUsage.tsx

```diff
@@ -390,8 +362,6 @@ const OldNetworkTrafficUsage: FC = () => {
         toChartData,
         toTrafficUsageSum,
         endpointsInfo,
-        calculateOverageCost,
-        calculateEstimatedMonthlyCost,
```

Comment:
These two functions are now imported directly from the neighboring traffic calculations file. The functions have been copied over directly along with their tests, so I'm not worried about this change.